lib/src/widgets/picker.dart

```diff
@@ -36,10 +41,15 @@ class Picker<Item> extends StatefulWidget {
   final String? description;
   final Function(Item) onItemSelected;
   final MainAxisAlignment mainAxisAlignment;
-  final String Function(Item)? displayItem;
+  final String Function(Item,{int? customValue})? displayItem;
```

Comment:
I don't like this, I believe there is a better way to handle this without having to add this parameter all over the code, when it's un-needed

Reply:
fixed